### PR TITLE
[PG-919] Risk Words

### PR DESCRIPTION
**Pull request by avi-bh**
### JIRA Ticket
https://bighealth.atlassian.net/browse/PG-919

### Problem
As a user happiness officer, I want to know when certain risk words are used in the Community app by users.

### Solution
* Submit a request to the Risk Words service when a new post is created.
* If that service detects risk words, email communityadmin@sleepio.com with details.

#### Unit tests + Integration Tests
* Added unit test for post middleware.

#### Manual Tests
* Manually tested in dev (and you can too at https://community.sleepiodev.com).

#### Related PRs
https://github.com/sleepio/platform-service-cluster/pull/172

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -105,6 +105,7 @@ settings/staff_web.db
 bh
 bh_settings
 data/*
+avatars/*
```
Comment (wsbastian) [inline .gitignore:108]: omg yes.

File: webapp/src/community_app/settings/local.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@ use_remote_dynamic_settings: false
 
 DEBUG: true
 
-secure_cookies: false  # Enable if using a proxy like ngrok
+secure_cookies: true # Enable if using a proxy like ngrok
```
Comment (wsbastian) [inline webapp/src/community_app/settings/local.yml:5]: can we run local with ngrok, given the cross subdomain cookie reqs?

**Review by wsbastian — COMMENTED** [review]
nice stuff!

File: webapp/src/community_app/settings/local.yml (hunk 2 above)
Comment (avi-bh) [reply webapp/src/community_app/settings/local.yml:5]: my bad, i thought this was client gateway

**Review by wsbastian — APPROVED** [review]
LGTM

**Review by wsbastian — APPROVED** [review]
Good catch, LGTM.

**Review by wsbastian — APPROVED** [review]

**Pull request merged**